src/internal/operators/distinctUntilChanged.ts

```diff
@@ -127,17 +126,16 @@ export function distinctUntilChanged<T>(comparator?: (previous: T, current: T) =
  * // { updatedBy: 'blesh', data: Array[0] }
  * ```
  *
+ * @see {@link distinct}
+ * @see {@link distinctUntilKeyChanged}
+ *
  * @param comparator A function used to compare the previous and current keys for
  * equality. Defaults to a `===` check.
  * @param keySelector Used to select a key value to be passed to the `comparator`.
+ *
  * @return A function that returns an Observable that emits items from the
  * source Observable with distinct values.
  */
-export function distinctUntilChanged<T, K>(
-  comparator: (previous: K, current: K) => boolean,
-  keySelector: (value: T) => K
-): MonoTypeOperatorFunction<T>;
```

Comment:
Moved to [top](https://github.com/ReactiveX/rxjs/pull/6756/files#diff-9be4aaa20e2c54b85cde0eb41759c61348e0868293683c9e836107120c0b103dR6-R10).